stories/CustomToolbar/index.js

```diff
@@ -38,7 +38,10 @@ const CustomToolbar = () =>
       toolbarClassName="rdw-storybook-toolbar"
       wrapperClassName="rdw-storybook-wrapper"
       editorClassName="rdw-storybook-editor"
-      toolbarCustomButtons={[<CustomOption />]}
+      toolbarCustomButtons={[<CustomOption />, {
+        component: <CustomOption />,
+        positionAfter: 'fontSize',
+      }]}
```

Comment:
Why you need to insert component in array and again in object inserting same component? That can be simplified.